src/xcode/ENA/ENA/Source/Scenes/ContactDiary/DiaryCoordinator.swift

```diff
@@ -65,30 +65,32 @@ class DiaryCoordinator {
 
 	/// Directly open the current day view. Used for deep links & shortcuts
 	func showCurrentDayScreen() {
-		// Info view MUST be shown
-		guard infoScreenShown else {
-			Log.debug("Diary info screen not shown. Skipping further navigation", log: .ui)
-			// set this to true to open current day screen after info screen has been dismissed
-			showCurrentDayScreenAfterInfoScreen = true
-			return
-		}
-
-		// check if the data model is correct
-		let model = DiaryOverviewViewModel(
-			diaryStore: diaryStore,
-			store: store,
-			homeState: homeState
-		)
-		guard let today = model.days.first else {
-			Log.warning("Can't get 'today' from `DiaryOverviewViewModel`. Discarding further quick action handling.", log: .ui)
-			return
-		}
 		// prevent navigation issues by falling back to overview screen
 		viewController.popToRootViewController(animated: false)
 		// dismiss all presented view controllers
 		viewController.view.window?.rootViewController?.dismiss(animated: false, completion: { [weak self] in
+			guard let self = self else {
+				return
```

Comment:
Some Log would be nice here

Reply:
Do we need a log if this view controller was released from memory?